docs/application/native/guides/multimedia/media-conversions.md

```diff
@@ -1,18 +1,14 @@
 # Media Conversions
 
 
-You can perform various media conversions through codecs. You can encode and decode video and audio data, and, in mobile applications, you can also use video transcoding features, such as multi-format codec and output format.
+You can perform various media conversions through codecs. You can encode and decode video and audio data, and, in mobile applications.
```

Comment:
Please remove:      , and, in mobile applications

Change to:
You can perform various media conversions through codecs. You can encode and decode both video and audio data.